designs/ProjectBoards.md

```diff
@@ -0,0 +1,32 @@
+# Using GitHub Project Boards for Customer Visibility and Tracking
+
+Our project boards will need to be created and accessed at the organizational level, not in any one repo (i.e. not in Home), because this is a restriction on project boards--if they're in a repo, they will be locked only to issues from that repo. Here's where ours will live: [https://github.com/orgs/NuGet/projects](https://github.com/orgs/NuGet/projects)
```

Comment:
Projects should be inside NuGet/Home so the public can see them.

Reply:
![image](https://user-images.githubusercontent.com/1288848/106147253-4432a080-613d-11eb-8ff5-083e366a2150.png)

Reply:
This won't work. If they're in Home they can only show Home repo items.

Reply:
ah crap, how do we make them public? Can we?

Reply:
Yes--it's a setting: public/private. I just made this one public briefly to test: 
![image](https://user-images.githubusercontent.com/14225979/106216415-c8108980-621e-11eb-9684-96f6301697b4.png)

Reply:
I've checked this in the past with InPrivate browser, and I could see 3 projects. I still see 2 (not Drew's). 
Trying to figure out how Jon is seeing none.
![image](https://user-images.githubusercontent.com/49205731/106486950-3011da80-6480-11eb-813f-3b8e39ff23f0.png)

Reply:
@donnie-msft at the time of my comment, all org projects are marked private. Drew mentioned that you can see them publicly by switching them over. Btw this is for org level not for home level. The limitation of home repo seems to be that we can't include gallery or client repos inside.

https://github.com/orgs/NuGet/projects

Reply:
Ok, this thread is confusing, but just wanted to point out Home projects have been public since at least October. But sounds like we wanted orgs for other reasons than that, which is fine

Reply:
Basically a home only project would only allow home issues.

An org project would allow issues from all repos (even private)